eureka-client/src/main/java/com/netflix/discovery/NotImplementedRegistryImpl.java

```diff
@@ -12,11 +12,11 @@
 
     @Override
     public Applications fetchRegistry() {
-        throw new UnsupportedOperationException("Backup registry not implemented.");
+        return null;
```

Comment:
As we change the behavior lets specify the contract in more details in `BackupRegistry` class.